flink-connector-mysql-cdc/src/test/java/com/ververica/cdc/connectors/mysql/source/NewlyAddedTableITCase.java

```diff
@@ -151,6 +151,20 @@ public void testNewlyAddedTableForExistsPipelineTwiceWithAheadBinlog() throws Ex
                 "address_shanghai");
     }
 
+    @Test
+    public void testNewlyAddedTableForExistsPipelineTwiceWithAheadBinlogAndAutoCloseReader()
+            throws Exception {
+        testNewlyAddedTableOneByOne(
+                DEFAULT_PARALLELISM,
+                ", 'scan.incremental.close-idle-reader.enabled'='true'",
```

Comment:
sourceOptions should be a `Properties` or `Map`, instead of a `String` specific format